source/smart-contracts/tutorials/piggy-bank/writing.rst

```diff
@@ -261,40 +258,23 @@ a mutable reference to the current state of the instance:
       todo!("Implement")
    }
 
-The macro requires the name of the contract given using the ``contract``
-attribute, which should match the name in the corresponding attribute in |init|_
+The ``contract`` attribute supplies the name of the contract to the macro.
+This name should match the name in the corresponding attribute in |init|_
 (``"PiggyBank"`` in our case). It also requires a name to identify this
-particular receive function using ``name``, this name together with the
-contract name have to be unique for a smart contract module.
+particular receive function using ``name``. The name and contract attributes
+each have to be unique within a smart contract module.
 
 The return type of the function is ``ReceiveResult<A>``, which is an alias for
 ``Result<A, Reject>``.
-Here ``A`` implements |HasActions|, which exposes functions for creating
-different actions.
-
-.. rubric:: Actions
+Here, ``A`` implements |HasActions|, which exposes functions for creating
+various :ref:`actions<action-descriptions>`.
 
-A smart contract can produce 3 types of actions:
-
-- **Accept**: Accept incoming GTU. Always succeeds.
-- **Simple Transfer**: Transfer some amount of GTU from the balance of the
-  smart contract instance to an account.
-- **Send**: Trigger receive function of a smart contract instance, with
-  a parameter and an amount of GTU.
-
-Additionally there are two ways to sequence these actions:
-
-- **And**: Runs the first action, if it succeeds runs the second action,
-  otherwise results in rejection.
-- **Or**: Runs the first action, **if it fails**, runs the second action,
-  otherwise results in success.
-
-In this contract we will only need to use **Accept** and **Simple Transfer**.
+In this contract we will use the **Accept** and **Simple Transfer** actions.
 
 Inserting GTU
 -------------
 
-The first interaction we will specify for our piggy bank is how to insert GTU.
+The first interaction we will specify for our piggy bank is how to put money into it.
```

Comment:
```suggestion
The first interaction we will specify for our piggy bank is how to insert GTU into it.
```
Following the assumption that we should say GTU instead of `money`